src/actions/transformCaseAction.ts

```diff
@@ -22,37 +22,37 @@ module Plywood {
 
     static fromJS(parameters: ActionJS): TransformCaseAction {
       var value = Action.jsToValue(parameters);
-      value.transformCaseType = parameters.transformCaseType;
+      value.transformType = parameters.transformType;
```

Comment:
this ether needs a back compat assignment if we want to change the name "transformCaseType" to "transformType"

Something like:

```
value.transformType = parameters.transformType || (parameters as any).transformCaseType;
```